terraso_backend/apps/core/models/users.py

```diff
@@ -114,6 +115,36 @@ def soft_delete_policy_action(self, **kwargs):
     def is_group_manager(self, group_id):
         return self.memberships.managers_only().filter(group__pk=group_id).exists()
 
+    def full_name(self):
+        return f"{self.first_name} {self.last_name}".strip()
+
+    def name_and_email(self):
+        return f"'{self.full_name()}' <{self.email}>"
+
+    def notifications_enabled(self):
+        preferences = self.preferences.filter(key="notifications")
+        if len(preferences) != 1:
+            return False
+
+        try:
+            return preferences[0].value.lower() == "true"
+        except AttributeError:
+            return False
+
+    def language(self):
+        preferences = self.preferences.filter(key="language")
+        if len(preferences) != 1:
+            return settings.LANGUAGE_CODE
+
+        try:
+            language_code = preferences[0].value
+            if language_code[0:2] in [lang[0] for lang in settings.LANGUAGES]:
```

Comment:
Suggestion: The system should handle locales not just languages, there can be fallbacks defined but in general just handling languages is not enough. Maybe a comment at this stage to not lose the future improvement can be enough?

Reply:
I think it is a broader issue, I see that the po files are related to language and not locales, so eventually when we want to have more granularity here we will have to refactor it, but this can be a future improvement since the locale is in the language_code